local/run.sh

```diff
@@ -62,7 +62,7 @@ EOF
       echo "Successfully contacted cluster with $MYSQL_INNODB_CLUSTER_MEMBERS members. Bootstrapping."
     fi
     echo "Succesfully contacted mysql server at $MYSQL_HOST. Trying to bootstrap."
-    mysqlrouter --bootstrap "$MYSQL_USER@$MYSQL_HOST:$MYSQL_PORT" --user=mysqlrouter --directory /tmp/mysqlrouter --force < "$PASSFILE"
+    mysqlrouter --bootstrap "$MYSQL_USER@$MYSQL_HOST:$MYSQL_PORT" --user=mysqlrouter --directory /tmp/mysqlrouter --conf-set-option=http_server.bind_address=127.0.0.1 --force < "$PASSFILE"
```

Comment:
Does this option applies only when bootstrapping, i.e. is written to the config file?